app/knowledge_panels.py

```diff
@@ -39,3 +41,141 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def data_quality_kp(
+    facet,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "country":
+        path = f"{facet}/{value}"
+        description = f"{value}"
+    else:
+        if country is not None:
+            path = f"country/{country}/{facet}/{value}"
+            description = f"{facet} based for {country}"
+        else:
+            path = f"{facet}/{value}"
+            description = f"{value} based for {facet}"
+    url = "https://world.openfoodfacts.org/"
+    source_url = urljoin(url, path)
+    description = f"data-quality issues related to {description}"
+    quality_url = f"{source_url}/data-quality.json"
+    """
+    Parsing data from the url
+    """
+    response_API = requests.get(quality_url)
+    data = response_API.json()
+    total_issues = data["count"]
+    # Returns total number of issues
+    tags = data["tags"]
+    first_three = tags[0:3]
+    # Returns First three issues sorted by most common
+    id = []
+    name = []
+    products = []
+    known = []
+    url = []
+    for i in first_three:
+        id.append(i["id"])
+        name.append(i["name"])
+        products.append(i["products"])
+        known.append(i["known"])
+        url.append(i["url"])
+    # Fetching data from first_three(sorted first three data)
```

Comment:
There is no added value to do this. On the contrary the previous structure is far better.